scripts/trifingerpro_post_submission.py

```diff
@@ -15,21 +15,107 @@
 import numpy as np
 import pandas
 import rospkg
+import yaml
 
+import robot_interfaces
 import robot_fingers
 import trifinger_object_tracking.py_tricamera_types as tricamera
 
 
+def get_robot_config(position_limits=True) -> str:
+    robot_config_file = "/etc/trifingerpro/trifingerpro.yml"
+
+    if position_limits:
+        return robot_config_file
+    else:
+        with open(robot_config_file, "r") as fh:
+            config = yaml.load(fh)
+
+        # remove position limits
+        config["hard_position_limits_lower"] = [-np.inf] * 9
+        config["hard_position_limits_upper"] = [+np.inf] * 9
+        del config["soft_position_limits_lower"]
+        del config["soft_position_limits_upper"]
+
+        tmp_config_file = "/tmp/trifingerpro.yml"
+        with open(tmp_config_file, "w") as fh:
+            yaml.dump(config, fh)
+
+        return tmp_config_file
+
+
+def end_stop_check(robot):
+    # Distance from the zero position (finger pointing straight down) to the
+    # end-stop.  This is independent of the placement of the encoder disc and
+    # thus should be the same on all TriFingerPro robots.
+    # TODO: this should actually be the same for all three fingers!  Needs to
+    #       be fixed in the calibration script, though.
+    zero_to_endstop = np.array(
```

Comment:
I'd find it a bit better if all hard coded variable were to be defined and declared private at the beginning of the file,
e.g.

```python
# beginning of file
_homing_torque = [+0.3, +0.3, -0.2] * 3
# in function
global _homing_torque
action = robot.Action(torque=_homing_torque)
```

Reply:
Done.  I realised that the homing torque and probably also the max. torque could actually be read from the config file, but this would require a bit restructuring, so I would not do this at the moment.